### PR TITLE
DutchX => dxDAO

### PR DESCRIPTION
**Pull request by dkent600**
also cleans up a bit the error messaging during UI loading

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -38,10 +38,12 @@ export class App {
   private intervalId: any;
 
   constructor(
-      private web3Service: Web3Service
+    // tslint:disable: align
+    private web3Service: Web3Service
```
Comment (orenyodfat) [inline src/app.ts]: why ?

File: src/organizations/dashboard.ts [hunk 2]
```diff
@@ -115,7 +115,8 @@ export class Dashboard {
     ]);
 
   constructor(
-      private daoService: DaoService
+    // tslint:disable: align
+    private daoService: DaoService
```
Comment (orenyodfat) [inline src/organizations/dashboard.ts]: why you modify that ?
why need to add // tslint:disable: align all over the code ?

**Comment by dkent600** [comment]
@orenyodfat 

> why need to add // tslint:disable: align all over the code ?

Yeah, this is annoying.  The tslint rules "indent" and "align" conflict when a method argument list looks like this:

```
myMethod(
  param1: type
  , param2: type
  , param3: type
)
```
tslint "align" wants this:

```
myMethod(
    param1: type
  , param2: type
  , param3: type
)
```

But my editor (vscode) insists on indenting according to the 2-spaces indenting rule.  This editor feature is far to useful to turn off.  But I also like the way of formatting method params with the comma in front.  It makes for easier editing of param lists and cleaner diffs.

An alternative to manually disabling the "align" rule in these specific cases would be to disable the rule globally for all method parameter lists.

My preference would be to disable the rule globally for all method parameter lists.  Do you care one way or the other, @orenyodfat ?

**Comment by dkent600** [comment]
> if you cannot controllable your editor preference so it is not so useful.

I can control it, but the ability to auto-correct indenting is extremely useful and no way am I willing to turn that off.  It was off only recently because it was inadvertantly turned off when I was working adding tslint.

> I prefer to set it back..as it was before this pr.

Putting the comma in front of the params is not uncommon.  I have worked with people who vehemently insisted on doing it this way, and I have come to appreciate that it really does present an easier way to work with adding and removing method parameters, and it yields a cleaner diff for PRs.

I don't think that looking "weird" (unfamiliar) is a strong reason for not using that format.  One gets used to it easily enough and comes to appreciate its usefulness.

Here is another alternative that I didn't suggest before because I only just thought of it:  that we make the  "align" rule, for method parameters only, a warning instead of as is, an error, and instead of turning it off altogether.

> Returning to what we had before

What we had before was no lint with my editor doing auto-indenting.  Then it was lint with no auto-indenting, but that is **really** no good for me.  So we need another option.  The ones I can think of:

* globally turn the align rule for method params into a warning
* globally disable the align rule for method params
* manually disable the align rule as I have done in this PR

**Comment by orenyodfat** [comment]
> > if you cannot controllable your editor preference so it is not so useful.
> 
> I can control it, but the ability to auto-correct indenting is extremely useful and no way am I willing to turn that off. It was off only recently because it was inadvertantly turned off when I was working adding tslint.
> 
> > I prefer to set it back..as it was before this pr.
> 
> Putting the comma in front of the params is not uncommon. I have worked with people who vehemently insisted on doing it this way, and I have come to appreciate that it really does present an easier way to work with adding and removing method parameters, and it yields a cleaner diff for PRs.
> 
> I don't think that looking "weird" (unfamiliar) is a strong reason for not using that format. One gets used to it easily enough and comes to appreciate its usefulness.
> 
> Here is another alternative that I didn't suggest before because I only just thought of it: that we make the "align" rule, for method parameters only, a warning instead of as is, an error, and instead of turning it off altogether.
> 
> > Returning to what we had before
> 
> What we had before was no lint with my editor doing auto-indenting. Then it was lint with no auto-indenting, but that is **really** no good for me. So we need another option. The ones I can think of:
> 
> * globally turn the align rule for method params into a warning
> * globally disable the align rule for method params
> * manually disable the align rule as I have done in this PR

if your (very specific) editor auto indent  is not align with tslint ..it is not useful.

**Comment by dkent600** [comment]
> if your (very specific) editor auto indent is not align with tslint ..it is not useful.

It is super useful in every other way!

One can also view this as tslint align and indent rules conflicting.

**Comment by dkent600** [comment]
I did some more research.  tslint does not auto-fix spacing to 2, it only aligns things.  Thus the editor is still the only option I know to auto-fix spacing.  vscode does it great, except in this one instance.

**Comment by dkent600** [comment]
@orenyodfat I think the best option here is to use the manual rule disabling as I have done in this PR.

1) it retains misalignment as an error, which is what we want
2) it avoids annoying warning messages that we'd get if making this just a warning
3) it is clear what is the intent of the coder
4) it retains the very useful option to use a tool such as vscode to auto-format spacing and alignment.
5) it retains the ability to use the `, paramName` pattern which is good for coding and diffing the addition and deletion of method parameters.

**Comment by orenyodfat** [comment]
> @orenyodfat I think the best option here is to use the manual rule disabling as I have done in this PR.
> 
> 1. it retains misalignment as an error, which is what we want
> 2. it avoids annoying warning messages that we'd get if making this just a warning
> 3. it is clear what is the intent of the coder
> 4. it retains the very useful option to use a tool such as vscode to auto-format spacing and alignment.
> 5. it retains the ability to use the `, paramName` pattern which is good for coding and diffing the addition and deletion of method parameters.

I am not agree.
please remove this linting staff from this pr it has nothing to do with the name of the dao .
You can open a new pr for linting , so we could discuss the linting issue there ..if you wish.

**Comment by dkent600** [comment]
@orenyodfat 

>You can open a new pr for linting , so we could discuss the linting issue there ..if you wish.

i changed all the commas back how they were

**Pull request merged**